src/rpc/handlers/AccountInfo.cpp

```diff
@@ -122,8 +123,9 @@ AccountInfoHandler::process(AccountInfoHandler::Input const& input, Context cons
             if (!signersKey.check(sleSigners))
                 return Error{Status{RippledError::rpcDB_DESERIALIZATION}};
 
-            out.signerLists = std::vector<ripple::STLedgerEntry>{sleSigners};
+            signerList.push_back(sleSigners);
         }
+        out.signerLists = std::move(signerList);
```

Comment:
We could avoid this by creating vector inside `out` at the line 116:
```cpp
out.signerList = std::vector<ripple::STLedgerEntry>();
```
And push into it on the line 126: 
```cpp
out.signerList->push_back(signerList);
```